EIPS/eip-867.md

```diff
@@ -0,0 +1,194 @@
+*We accidentally put the wrong contract address up on our website.  Can you please refund any eth sent to 0x1234 back to the senders.  Thanks.*
+
+**Very Bad example** (not objective, one person’s word against another):
+*I sent tokens to X for services and he did a lousy job.  I want my money back but he won’t refund me.  Please help!!*
+
+### Verification Script
+A machine readable script that outputs a single State Change Object. This script should be implemented so that it is easily audited by a reviewer. Verification scripts should be javascript files that may use the web3.js(https://github.com/ethereum/web3.js/) library. 
+
+There are a few guidelines for verification scripts:
+
+- Scripts should always be written to be as concise as reasonably possible, and anyone executing the verification script should review it first to verify that it does not contain any unsafe operations. 
+- No verification script should ever require an unlocked ethereum wallet 
+- The script hardcode the highest block included during execution (otherwise the results may differ between runs).
+
+The purpose of the ERP Verification script is to unambiguously specify (through code) the criteria used to compute the set of State Change Actions.  The script’s output, as described above, will be the input used by all Ethereum clients.  Client teams should avoid manually pre-processing the artifact or using the artifact to simply guide changes in the code.  Instead, the artifact should be bundled with the client and processed directly by the client at the specified block. This will minimize the amount of client effort required for each ERP and help to ensure compatibility between clients. We anticipate that some ERP Verification scripts may be trivial but we recommend their inclusion for consistency. 
+
+### State Change Object
+The State Change Object is a standard format that will be interpretable by all Ethereum clients.
+
+It is a single JSON object containing the following fields:
+- **erpId**: A string identifier for this ERP (likely the associated EIP number, e.g. “EIP-1234”).  This will converted from ascii to a hex string, then added as extra data on the target block.
+- **targetBlock**: The block at which the stateChange should be applied.  Clients would use this to determine when a set of state changes should occur
+- **actions**: An array of State Change Actions.
+- **metadata**
+  - **sourceBlock**: The highest block considered by the script when it was run.
+  - **version**: The version of the verification script when it was run.  
+
+### State Change Actions
+A JSON object that includes a “type” field and a set of “data” fields, where the contents of the data fields is dependent on the type and must follow the schema define for that type.  This allows clients to support a limited set of operations initially and add more based on a subsequent EIP if needed.  Support for the following action types should be implemented by clients upon adoption of this EIP:  
```

Comment:
- data fields _are_ dependant
- schema _defined_ for that type
- (first: sentence fragment - this sentence lacks subject/verb)